code_generation/code_generation.py

```diff
@@ -311,6 +311,8 @@ def write_code(self, calls: str, includes: str, run_commands: str) -> None:
                 .replace("{ANALYSISTAG}", '"Analysis={}"'.format(self.analysis_name))
                 .replace("{PROGRESS_CALLBACK}", self.set_process_tracking())
                 .replace("{OUTPUT_QUANTITIES}", self.set_output_quantities())
+                .replace("{SHIFT_QUANTITES_MAP}", self.set_shift_quantites_map())
```

Comment:
typo: quantities
this typo appears quit often and also in different files of this PR. Or is this intended?

Reply:
i hope i caught all occasions of this typos